runtime/parachains/src/paras_inherent.rs

```diff
@@ -1191,33 +1186,33 @@ mod tests {
 		}
 
 		#[test]
+		// Ensure that when a block is overlength due to disputes, but there is still sufficient
+		// block weight to include a number of signed bitfields, the inherent data is filtered
+		// as expected
 		fn limit_dispute_data_ignore_backed_candidates() {
 			new_test_ext(MockGenesisConfig::default()).execute_with(|| {
 				// Create the inherent data for this block
 				let dispute_statements = BTreeMap::new();
 
 				let mut backed_and_concluding = BTreeMap::new();
+				// 2 backed candidates shall be scheduled
 				backed_and_concluding.insert(0, 2);
 				backed_and_concluding.insert(1, 2);
 
 				let scenario = make_inherent_data(TestConfig {
 					dispute_statements,
-					dispute_sessions: vec![0, 0, 2, 2, 1],
+					dispute_sessions: vec![0, 0, 2, 2, 1], // 2 backed candidates, 3 disputes at sessions 2, 1 and 1 respectively
```

Comment:
this comment should be moved above this assignment - confusing why its right here